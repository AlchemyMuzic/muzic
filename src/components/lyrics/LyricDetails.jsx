import React from 'react';

export default function Lyrics() {
  return (
    <div>
        Lyrics Component
    </div>
  );
}
