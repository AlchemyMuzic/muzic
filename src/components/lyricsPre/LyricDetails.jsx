
// import React from 'react';
// import PropTypes from 'prop-types';
// import { Link } from 'react-router-dom';

// export default function ArtistAlbumDetails({ releases }) {

//   const releasesElements = releases.map(release => (
//     <li key={release.release_id}>
//       <Link to={`/releases/${release.id}`}>
//         <h2>{release.album}</h2>
//         <img
//           src={`http://coverartarchive.org/release/${release.release_id}/front`}
//           alt="release.album"/>
//       </Link>
//     </li>
//   ));

//   return (
//     <ul data-testid="releaseTest">
//       {releasesElements}
//     </ul>
//   );
// }

// ArtistAlbumDetails.propTypes = {
//   releases: PropTypes.arrayOf(PropTypes.shape({
//     release_id: PropTypes.string.isRequired,
//     album: PropTypes.string.isRequired
//   }))
// };
