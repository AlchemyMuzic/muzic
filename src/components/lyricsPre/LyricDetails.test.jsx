import React from 'react';
import { render, cleanup } from '@testing-library/react';
import LyricDetails from './LyricDetails';

describe('LyricDetails component', () => {
    afterEach(() => cleanup());
    it('renders a list of artists', () => {
        const { asFragment } = render(<LyricDetails
            lyrics={''} />);
        expect(asFragment()).toMatchSnapshot();
    })
})
