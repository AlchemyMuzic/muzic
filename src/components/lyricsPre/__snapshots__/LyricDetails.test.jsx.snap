// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LyricDetails component renders a list of artists 1`] = `
<DocumentFragment>
  <p
    data-testid="lyricsTest"
  />
</DocumentFragment>
`;
